### PR TITLE
Allow Welcome Message on Room Creation

### PR DESCRIPTION
**Pull request by dangrous**
### Details

You'll get the "too many auth calls" alert for this, due to the welcome message being added as an NVP. @jasperhuangg let me know how you want to handle this, whether we just add it to the list of exceptions, or if we need to somehow roll it into Auth. **For local testing, please add the command to [the list of exceptions](https://github.com/Expensify/Web-Expensify/blob/52f4c4df17189a586566e960cf563ca1a62e5e71/lib/APIWriteCommands.php#L91) otherwise you'll get an error when you try to create the room and won't be able to test. Staging server should be fine.**
 
This also covers a bit of a refactor for the room creation page, using push-to-page selectors. Since we didn't really have a generic one of these before for use in forms, I created one. It takes a list of items (with values and labels) and will make them into one of these pop up selection modals. Handles display of the selected item, and offers an option for an initial placeholder (though you can't get back to it afterwards, which isn't an issue here since selections are required for all).

I'm not 100% confident on the name of the new component, so any suggestions are welcome.

Lastly, I also had to update the NewChatSelectorPage to include `withOnyx` because otherwise it wasn't actually able to access the betas prop, but now it works.

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/316189
PROPOSAL: N/A

### Tests

* Same as QA

- [X] Verify that no errors appear in the JS console

### Offline tests

* Same as QA (and confirm that the room is correctly created with no errors once you return online)

### QA Steps

1. Log into an account that has at least two workspaces (or create them)
2. Open the FAB
3. Click "Send message"
4. Toggle over to "Room"
5. Enter a room name
6. Ensure that an error is shown if you do NOT enter a room name
7. Enter a welcome message
8. Ensure that the form shows an error if you do not select a workspace (and try to create room)
9. Select a workspace - ensure that all possible workspaces are in the list and selectable
10. Ensure that your selection persists in the main form
11. If you are an admin of a workspace, ensure that the "Who can post" selector appears once you have chosen that workspace.
12. Test that all options for "Who can post" are selectable and persist to the main form
13. Test that all options for "Visibility" are selectable and persist to the main form
14. Submit the form and ensure that the room is created as expected with the correct qualities.
15. Log in to another account and navigate to that room (assuming it is public).
16. Ensure you are sent the welcome message you entered in step 6.

- [X] Verify that no errors appear in the JS console

### PR Author Checklist

- [X] I linked the correct issue in the `### Fixed Issues` section above
- [X] I wrote clear testing steps that cover the changes made in this PR
    - [X] I added steps for local testing in the `Tests` section
    - [X] I added steps for the expected offline behavior in the `Offline steps` section
    - [X] I added steps for Staging and/or Production testing in the `QA steps` section
    - [X] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [X] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [X] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [X] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I ran the tests on **all platforms** & verified they passed on:
    - [X] Android / native
    - [X] Android / Chrome
    - [X] iOS / native
    - [X] iOS / Safari
    - [X] MacOS / Chrome / Safari
    - [X] MacOS / Desktop
- [X] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [X] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [X] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [X] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [X] I verified that comments were added to code that is not self explanatory
    - [X] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [X] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [X] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [X] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [X] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [X] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [X] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [X] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [X] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [X] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [X] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [X] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [X] I verified that if a function's arguments changed that all usages have also been updated correctly
- [X] If a new component is created I verified that:
    - [X] A similar component doesn't exist in the codebase
    - [X] All props are defined accurately and each prop has a `/** comment above it */`
    - [X] The file is named correctly
    - [X] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [X] The only data being stored in the state is data necessary for rendering and nothing else
    - [X] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
    - [X] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [X] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [X] All JSX used for rendering exists in the render method
    - [X] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [X] If any new file was added I verified that:
    - [X] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [X] If a new CSS style is added I verified that:
    - [X] A similar style doesn't already exist
    - [X] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [X] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [X] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [X] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [X] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [X] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [X] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [X] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

Uploading web.mp4…

</details>

<details>
<summary>Mobile Web - Chrome</summary>

I can't get this to login, on `main` either, so I'm forgoing this. Some combo of Bali + ngrok + cloudflare + etc. etc.. If you could double test this it would be appreciated!

</details>

<details>
<summary>Mobile Web - Safari</summary>

https://github.com/Expensify/App/assets/5487802/0f60e678-5465-4bf8-9e35-86a5ff82f0a0

</details>

<details>
<summary>Desktop</summary>

https://github.com/Expensify/App/assets/5487802/82b0b06a-c20b-4cc4-b1a4-de36e5974ea7

</details>

<details>
<summary>iOS</summary>

https://github.com/Expensify/App/assets/5487802/9b01da80-ada8-4448-bbe0-e57e7aeae94e

</details>

<details>
<summary>Android</summary>

This is coming! Taking forever to build so I wanted to get a head start.

</details>

### CODE REVIEW

**Comment by shawnborton** [comment]
Just a note that I think we want `mb5` for our form spacing for this one, cc @Expensify/design

**Review by eVoloshchak — CHANGES_REQUESTED** [review]

**Comment by eVoloshchak** [comment]
Besides the two small comments above everything else looks good. I can't test the actual functionality of this until BE change is deployed, but all the FE changes work as expected

**Comment by dangrous** [comment]
@eVoloshchak BE change is on staging now so you should be able to test! 

Will review your feedback in a couple mins but wanted to call that out.

File: src/pages/workspace/WorkspaceNewRoomPage.js [hunk 1]
```diff
@@ -176,27 +177,41 @@ function WorkspaceNewRoomPage(props) {
...
-                            <View style={styles.mb2}>
-                                <Picker
+                            <View style={[styles.mb5]}>
+                                <TextInput
+                                    inputID="welcomeMessage"
+                                    label={translate('welcomeMessagePage.welcomeMessage')}
+                                    accessibilityLabel={translate('welcomeMessagePage.welcomeMessage')}
+                                    accessibilityRole={CONST.ACCESSIBILITY_ROLE.TEXT}
+                                    autoGrowHeight
+                                    placeholder={translate('welcomeMessagePage.welcomeMessage')}
```
[the remark this answers is not in the document]
Comment (dangrous) [reply src/pages/workspace/WorkspaceNewRoomPage.js]: whoops, not sure where that came from

File: src/pages/workspace/WorkspaceNewRoomPage.js [hunk 2]
```diff
@@ -176,27 +177,41 @@ function WorkspaceNewRoomPage(props) {
                                     autoFocus
                                 />
                             </View>
-                            <View style={styles.mb2}>
-                                <Picker
+                            <View style={[styles.mb5]}>
+                                <TextInput
+                                    inputID="welcomeMessage"
+                                    label={translate('welcomeMessagePage.welcomeMessage')}
```
[the remark this answers is not in the document]
Comment (dangrous) [reply src/pages/workspace/WorkspaceNewRoomPage.js]: yeah makes sense to me

**Comment by dangrous** [comment]
Updated!

**Comment by eVoloshchak** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android / native
    - [ ] Android / Chrome
    - [ ] iOS / native
    - [ ] iOS / Safari
    - [ ] MacOS / Chrome / Safari
    - [ ] MacOS / Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>



</details>

<details>
<summary>Mobile Web - Chrome</summary>



</details>

<details>
<summary>Mobile Web - Safari</summary>



</details>

<details>
<summary>Desktop</summary>



</details>

<details>
<summary>iOS</summary>



</details>

<details>
<summary>Android</summary>



</details>

**Review by eVoloshchak — COMMENTED** [review]
1. Bug on mWeb Safari:

If you're focused on Welcome Message input, there is a padding above the keyboard (it disappears a couple of seconds after the keyboard is closed)


https://github.com/Expensify/App/assets/9059945/061b2ac6-8cdd-4528-ba6c-adaea3c70150


2. Bug on native iOS (isn't present on Android)

The whole page re-renders (sometimes endlessly) after you select a workspace/who can post/visibility

https://github.com/Expensify/App/assets/9059945/d33125e4-e3f3-4631-b0ff-ad50026dbe3f

**Comment by dangrous** [comment]
eep. looking into these!

**Comment by dangrous** [comment]
Okay so for #1 I think we have to modify this line: https://github.com/Expensify/App/blob/bcbd156f73c5833f317f6dc32ed03f0246b49618/src/pages/workspace/WorkspaceNewRoomPage.js#L161

But I'm not quite sure how yet.

Working on 2 as well, but let me know if you have any ideas

**Comment by dangrous** [comment]
okay so that padding is because, when you focus the second input, it shifts the whole screen up, which hides part of the header/tab selector. When part of that is off the screen, the offset is now too big because it's accounting for the full height of the header, not just the visible part. @MaciejSWM I'm wondering if you might have any ideas on how to handle this since you wrote this part originally?

**Comment by dangrous** [comment]
Okay I pushed a change that might (?) fix the re-renders you saw @eVoloshchak . I couldn't replicate that locally, so you'll have to tell me if it works. I think it should? Stores more in state.

Let me know if the padding is working better without `scrollContextEnabled`, too. I don't THINK it will be, despite it making sense code wise, but just in case. I'll keep looking too.

**Comment by eVoloshchak** [comment]
@dangrous, both of the problems are still present
Also verified this isn't an emulator problem, physical device also has those problems

I'll try to test this on iOS17 a bit later, not that it should matter, but better be sure

**Comment by dangrous** [comment]
weird! I tried on a physical device (iOS17) and an emulator and couldn't get the rerendering.

**Comment by dangrous** [comment]
@eVoloshchak while we're investigating/trying to reproduce the re-rendering on our end, do you think you could try to figure out what's going on on your side? Would definitely be great to have your help!

**Comment by eVoloshchak** [comment]
Huh, I can't reproduce the re-rendering issue anymore
Yesterday I've updated to the latest version of MacOS and updated Xcode from 14 to 15, I can't fathom how any of those updates could have fixed that, but it is resolved 🧐
This is good, but confusing

**Comment by dangrous** [comment]
Okay, that's good (mostly, haha) to hear about the re-rendering. So strange!

As for the padding, I've determined that we need to do something in the `NewChatSelectorPage` component, either prevent scrolling altogether (which might be my vote, but I'm not sure how) or pass something down on a scroll event to WorkspaceNewRoomPage to allow it to modify https://github.com/Expensify/App/blob/25e3bfad4e4d66199d7ffbe7c1fd7f7b53b2156b/src/pages/workspace/WorkspaceNewRoomPage.js#L163

**Comment by eVoloshchak** [comment]
Looks like [shouldEnableMaxHeight](https://github.com/Expensify/App/blob/da07655c618299f330e18b48b18ba2f752f762aa/src/pages/NewChatSelectorPage.js#L33) might be at fault too. If you set it to `false`,  the padding disappears, but the scrolling is still kind of broken 

https://github.com/Expensify/App/assets/9059945/adf9e97d-0194-457d-9340-2fc7fa7de29d

I wonder why this bug is present only on mobile Safari. I'll try to dig into this more tomorrow

**Comment by dangrous** [comment]
Okay so I think I'm almost there. If you remove `shouldEnableMaxHeight` and `shouldEnableKeyboardAvoidingView={false}` from the `ScreenWrapper` in `NewChatSelectorPage`, and then add back `scrollContextEnabled` into the `Form` of `WorkspaceNewRoomPage` then it seems to work as expected except there's a bunch of empty space underneath the submit button of the room tab, and under the chat list of the Chat tab.

https://github.com/Expensify/App/assets/5487802/38c83ac2-6bb7-4b73-b96c-87a01375f2da

**Comment by dangrous** [comment]
Okay good/bad - there's actually already an existing bug with this weird space. If you have a long list of contacts on the `Chat` tab, the page is already cut off too high:
image
So I don't like it BUT I think we might be able to merge without fixing the padding since I'm guessing it's the same source. And then we can handle that separately, in the interest of closing this one out.

What do you think @eVoloshchak cc @MitchExpensify

**Comment by dangrous** [comment]
Okay confirmed in Slack - we're going to move forward and handle that padding bug separately (reported [here](https://expensify.slack.com/archives/C049HHMV9SM/p1695976040947749)). Let us know if you can do a final review with that in mind, @eVoloshchak - thank you!

**Comment by dangrous** [comment]
Can you double check that that doesn't happen on staging?

I'm not 100% sure it's a bug - like, it's silly to do so, but I guess there's no reason you can't create a room you can't post in - but maybe we should revert the writeCapability back to the default if isPolicyAdmin is false for the selected policy?

**Comment by dangrous** [comment]
Ah, it doesn't. Will fix!

**Review by eVoloshchak — APPROVED** [review]
Went through the checklist once more, LGTM

The re-rendering problem on iOS came back, but at this point I'm not sure if this is a legitimate bug or a problem with my ENV, since it's not reproducible by others and is not 100% reproducible by me, so approving.
We could fix this in a follow-up PR if QA will experience the same

**Review by marcochavezf — APPROVED** [review]
The code LGTM, waiting for @jasperhuangg if has some opinion about the "too many auth calls" alert. Or create a follow-up PR to add it to the whitelist and feel free to merge.

**Comment by dangrous** [comment]
I'm gonna merge and create a quick followup issue for the Auth calls thing

File: src/pages/workspace/WorkspaceNewRoomPage.js [hunk 3]
```diff
@@ -86,9 +88,17 @@ function WorkspaceNewRoomPage(props) {
...
+        Report.addPolicyReport(policyID, values.roomName, visibility, policyMembers, writeCapability, values.welcomeMessage);
     };
 
+    useEffect(() => {
+        if (isPolicyAdmin) {
+            return;
+        }
+
+        setWriteCapability(CONST.REPORT.WRITE_CAPABILITIES.ALL);
+    }, [policyID, isPolicyAdmin]);
```
Comment (mountiny) [inline src/pages/workspace/WorkspaceNewRoomPage.js:100]: the policyID here is not needed

Comment (dangrous) [reply src/pages/workspace/WorkspaceNewRoomPage.js:100]: ah, yep. i think that was from before I made it use isPolicyAdmin. Can update in a followup if needed!

**Pull request merged**